samples/app-hello-world/nodejs/.gitignore

```diff
@@ -64,4 +64,24 @@ build/
 
 # VS Code config files
 .vscode/**
```

Comment:
Need add `vscode` folder into source control.

Reply:
changed

Reply:
still not see .vscode folder?